common/src/main/java/org/cloudfoundry/autosleep/config/Config.java

```diff
@@ -113,6 +113,12 @@
         String SECRET = "secret";
 
     }
+    
+    interface OrgEnrollmentParameters {
+        enum EnrolledState {
+            backoffice_enrolled, backoffice_opted_out
```

Comment:
> How different is auto-enrolled different from backoffice-enrolled?

You're right, the difference is small. Might be useful in a UI to distinguish auto-enrolled orgs from manually opted-in enrollments.
I'm fine with restoring to ``enrolled, backoffice_opted_out`` if this is your preference.